packages/config/config/devices/0x031e/lzw42.json

```
@@ -66,5 +66,16 @@
 		"exclusion": "Put your hub in exclusion mode and turn the power to the bulb on.  Your bulb will blink twice (2x) indicating it's in exclusion mode.  When exclusion is successful, it will blink one more time (1x) to confirm.  Your hub should say that the device is excluded.",
 		"reset": "You may power on/off the bulb 6x (between 0.5-2 seconds each time) or use a certified controller to remove the device from your network to factory default.  Only use this procedure in the event that the network primary controller is missing or otherwise inoperable.  Your bulb will flash twice to confirm factory reset.",
 		"manual": "https://products.z-wavealliance.org/ProductManual/File?folder=&filename=product_documents/3613/LZW42%20Manual.pdf"
+	},
+	"compat": {
+		// The device supports Color Switch V2, despite the Version query saying
```

Heh. In this case what you did should work, but requires a re-interview to take effect.
Can you complete the comment please?
```suggestion
		// The device supports Color Switch V2, despite only reporting support for V1
```

---

I'm gonna be honest, I don't know where the last half of the comment went when I commited that...